inst/tutorials/data_visualization_basic/data_visualization_basic.Rmd

```diff
@@ -0,0 +1,301 @@
+              # The cruises that took place in Februrary can be determined
+              # using the Date variable and functions that we won't discuss in
+              # these data science modules
+              Cruise %in% c(18, 30, 42, 54, 66, 80, 92))
+```
+
+## Introduction
+This tutorial provides guided, hands-on instruction to create and modify plots using the `ggplot2` package in R. It also includes plots of an ANOVA analysis.
+
+After this tutorial, you will be able to:
+
+* Create dot and box plots in `ggplot2`
+* Modify attributes of ggplots
+* Complete and interpret the output of ANOVAs in R
+
+## Setup
+Prior to starting this tutorial, please complete the *Pre-module download assignment* to obtain all the necessary software and data.
+
+Create a new script named "ANOVA". As before, you begin by loading your packages with `library()`.
+```{r message=FALSE, warning=FALSE}
+library(tidyverse)
+```
+
+## Explore the metadata
+In addition to measurements of microbial communities, you also have geochemical data for the Saanich Inlet samples. For a brief introduction to these data, see Hallam SJ et al. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data 4: 170158 [doi:10.1038/sdata.2017.158](https://www.nature.com/articles/sdata2017158). 
+
+A subset of these data has been provided in `4.MICB301_stats_extend_data.csv`  on Canvas including:
+
+  - Depth: depth in meters
+  - In micromolar (uM) or nanomolar (nM)
+	- NO3_uM: nitrate 
+	- NCTD_O2: nitrite
+	- N2O_uM: nitrous oxide
+	- NH4_uM: ammonium
+	- H2S_uM: hydrogen sulfide
+	- CTD_O2: oxygen
+	- CH4_uM: methane
+
+Read `4.MICB301_stats_extend_data.csv` into R using `read_csv` and save as `raw_dat`.
+```{r}
+raw_dat <- geochemicals
+```
+
+## Data cleaning
+
+A data structure we haven't utilized in R yet are factors, which are used to represent categorical variables. In general, categorical variables can take on fixed number of possible values. For example, `Cruise` is a numeric right now, but actually represents a category (i.e. each cruise just indicates a group of measurements). To convert a vector to factors we use the `as.factors()` function. 
+```{r}
+dat <- mutate(raw_dat, Depth = as.factor(Depth),
+              Cruise = as.factor(Cruise))
+```
+
+```{r factors, echo=FALSE}
+question("Which of these variables could potentially be represented as categorical vectors? (select ALL that apply)",
+  answer("Colour of flower petals of 9 different plants", correct = TRUE),
+  answer("Oxygen concentration in uM"),
+  answer("Ammonium concentration in nM"),
+  answer("Number of days (3-10) it takes a plant to sprout", correct = TRUE),
+  incorrect = "Incorrect. Hint: Categorical variables usually take on a fixed set of values")
+```
+
+Using the `%in%` binary operator we can filter for groups of values. Subset data to 3 depths in 7 cruises (i.e. specimens) in February:
+```{r}
+dat <- filter(dat, Depth %in% c(10, 100, 200),
```

Comment:
I think it would be nice to have this part be an interactive exercise (assuming we expect students to know basic data wrangling skills)